### DISCUSSION TITLE
透過影片訓練AI玩Minecraft

### DISCUSSION THREAD

**Discussion by jerry0108566** [1101-GS3073-C/Python-Community-GS3073-C#731] [General]
原文：[https://www.ithome.com.tw/news/154487](https://www.ithome.com.tw/news/154487)
看影片他在20分鐘就做出鑽石稿了，我竟然連AI都不如...

**Comment by HsuChen88** [comment]
Minecraft可以結合這項技術，開放一個新功能，給懶惰的課長使用，他可以躺著領成果，也公司也可以賺課金的錢，一舉兩得。

**Comment by alvin1226** [comment]
如果讓AI機器學習速通minecraft，他會突破現在最快的紀錄嗎？那是當然，那會不會產生我們沒想到過的方法呢？

**Comment by ghost** [comment]
minecraft是一個極為自由且開放的遊戲，相對於其他遊戲有非常多的可能性，這也代表ai需要判斷的事情非常的多，如果真的可以靠這個將遊戲通關，是非常了不起的事情。
